api/KeyboardEvent.json

```
@@ -1441,7 +1441,7 @@
               "version_added": "5"
             },
             "safari_ios": {
-              "version_added": "5.1"
+              "version_added": "5"
```

Continuing from https://github.com/mdn/browser-compat-data/pull/11175#discussion_r655585705

It sounds like `keyCode` was in iOS 4.2 but always 0. If so I would still set this to 4.2, maybe it only worked with a keyboard attached, or is broken in the emulator. Without the ability to test, I'd prefer to assume the matching release:

```suggestion
              "version_added": "4.2"
```